modules/gitops-configuring-a-cluster-scoped-instance-to-manage-rollouts-resources.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-a-cluster-scoped-instance-to-manage-rollouts-resources_{context}"]
+= Configuring a cluster-scoped rollouts instance to manage rollouts resources 
+
+To configure a cluster-scoped rollouts instance for managing rollouts resources, add the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable in the `Subscription` resource. This variable contains a list of user-defined namespaces which can be configured for a cluster-scoped rollouts installation.
+
+[NOTE]
+====
+You can only create a cluster-scoped rollouts instance if the `NAMESPACE_SCOPED_ARGO_ROLLOUTS` variable is set to `false`.
+====
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Operators* → *Installed Operators* → *{gitops-title}* → *Subscription*.
+
+. Click the *Actions* list and then click *Edit Subscription*.
+
+. On the *openshift-gitops-operator* Subscription details page, under the *YAML* tab, edit the `Subscription` YAML file by adding the namespace of the Argo CD instance to the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable in the `spec` section:
```

Comment:
Should we also mention NAMESPACE_SCOPED_ARGO_ROLLOUTS variable here, since both are required to create cluster-scoped instance.

Reply:
I would like to leave it separate since we have already talked a lot about namespace-scoped installation in the previous section. Not fixed.